### ISSUE TITLE
Logwatcher attributes have zero values for some minutes after pimatic startup

### ISSUE THREAD

**Issue by sgoetze** [pimatic/pimatic#743]
I use log reader plugin to get sensor data from different sources (a RPi with some sensors, an ESP8266 with DS18B20) with nc. Received data are written to /tmp/netcat.log with "nc -d -k -l 8888 >>/tmp/netcat.log. A cron job stops nc once after midnight, cuts 15 lines from /tmp/netcat.log with the tail-command and creates a new /tmp/netcat.log with these lines. Then nc is started again.

There are many lines (15 after midnight, about 30 more every hour) with different values in netcat.log but they have only 4 different types: 
```
K,10.500000,41,12.100000,41,1
G,0.000000,11212.550000
W1,21.062000
W2,14.56
```
When I start or restart pimatic it is the normal situation that netcat.log exists and nc waits for the next data transfer.
In config.json is only one logwatcher device defined to get all these values.
```
    {
      "id": "nclog",
      "name": "nc-Logauswertung",
      "class": "LogWatcher",
      "file": "/tmp/netcat.log",
      "attributes": [
        {
          "name": "t_a",
          "type": "number",
          "unit": "°C"
        },
        {
          "name": "f_a",
          "type": "number",
          "unit": "%"
        },
        {
          "name": "t_k",
          "type": "number",
          "unit": "°C"
        },
        {
          "name": "f_k",
          "type": "number",
          "unit": "%"
        },
        {
          "name": "l_status",
          "type": "number"
        },
        {
          "name": "tick_gas",
          "type": "number",
          "unit": "l/15 min."
        },
        {
          "name": "zaehler_gas",
          "type": "number",
          "unit": "m³"
        },
        {
          "name": "it",
          "type": "number",
          "unit": "°C"
        },
        {
          "name": "wt",
          "type": "number",
          "unit": "°C"
        }
      ],
      "lines": [
        {
          "match": "^K,(.+),(.+),(.+),(.+),(.+)",
          "t_a": "$1"
        },
        {
          "match": "^K,(.+),(.+),(.+),(.+),(.+)",
          "f_a": "$2"
        },
        {
          "match": "^K,(.+),(.+),(.+),(.+),(.+)",
          "t_k": "$3"
        },
        {
          "match": "^K,(.+),(.+),(.+),(.+),(.+)",
          "f_k": "$4"
        },
        {
          "match": "^K,(.+),(.+),(.+),(.+),(.+)",
          "l_status": "$5"
        },
        {
          "match": "G,(.+),(.+)",
          "tick_gas": "$1"
        },
        {
          "match": "G,(.+),(.+)",
          "zaehler_gas": "$2"
        },
        {
          "match": "W1,(.+)",
          "it": "$1"
        },
        {
          "match": "W2,(.+)",
          "wt": "$1"
        }
      ]
    },
```
These 9 attributes are used in different variablesdevices.

Now the issue.
After picmatic has finished a startup the attributes of nclog-device have zero values. After some minutes (possibly after the first change of netcat.log - but I am not sure) the attributes change to the right values.
But the wrong zero values are logged to the database. I did restarts of pimatic many times and always with this result. So it's very improbable that netcat.log always had been locked by nc for a data transfer (if it is locked at all) during pimatic startup (in this case a warning could be written).

Especially for the graphs the zero values are a big problem. They are shown as glitches down to zero line. In the graphs zero is included even if all other values are bigger then 10000 (absolute counter of gas consumption - "zaehler_gas"). So all normal values are shown as a straight line (normal changes are much less, about 10-20). These changes cannot be seen on an Y-axis spreading from 0 ... 10000.

**Comment by sweetpi** [comment]
I've published pimatic-log-reader v0.8.7 which should fix the issue. Please reopen if the problem still exists.

**Comment by sgoetze** [comment]
It works. 

Thank you very much :-)
